doc/sphinxman/document_tests.pl

```diff
@@ -84,6 +84,7 @@
 foreach my $File(readdir SAMPLES){
     next unless $File =~ /\w+/; # Make sure we don't nuke .. or . !
     next if $File =~ /example_psi4rc_file/; # Keep the example psi4rc file
+    next if $File =~ /stdsuite_psi4.txt/;  # Keep the generated standard_suite record
```

Comment:
Background discussed https://github.com/psi4/psi4/pull/2731/files#diff-4a81ee0fb4705fe6acd6ac7f6417f5a899437a93780329a5f647a94eea425c24R8 . Any of docs/sphinxman/, samples/, or tests/pytests/ has a reasonable case for hosting the file. I settled on samples because that's where other regenerate-able stuff lives, but I don't really care.